app/src/containers/App.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```

Comment:
keep test files on you local. Don't push to git

Reply:
fix